modules/channel-slack/src/backend/index.ts

```diff
@@ -37,6 +37,7 @@ const onBotUnmount = async (bp: typeof sdk, botId: string) => {
     return
   }
 
+  client.shutdown()
```

Comment:
Missing an await here. VS Code should warn you about that, but the client was not typed properly. 

Can you add the await and add the correct typing in `typings.d.ts`  (any -> SlackClient)?